www/js/views/around.js

```diff
@@ -267,7 +268,6 @@
 
             onClickCancel: function(e) {
                 e.preventDefault();
-                fixmystreet.markers.removeAllFeatures();
```

Comment:
Not sure what removing this line does, is it done again elsewhere?

Reply:
Yep, the markers are refreshed on line 273. The previous behaviour would cause all the markers to disappear for a couple of seconds when returning to the map screen, which I thought was a bit confusing with the addition of the hide pins button. I couldn't see a reason for the markers to be programmatically removed before being refreshed from the server, but happy to put it back if there was one!

Reply:
I assume not – actually links with https://github.com/mysociety/fixmystreet/issues/2059 similar!